src/main/java/no/nav/pto/veilarbportefolje/opensearch/OpensearchQueryBuilder.java

```diff
@@ -209,6 +206,12 @@ static void leggTilManuelleFilter(BoolQueryBuilder queryBuilder, Filtervalg filt
             );
             queryBuilder.must(avvik14aVedtakSubQuery);
         }
+
+        if (filtervalg.harEnsligeForsorgereFilter()) {
+            if (filtervalg.getEnsligeForsorgere().contains(EnsligeForsorgere.OVERGANGSSTØNAD)) {
```

Comment:
Unnecessary nested `if` can be combined into single predicate.

Reply:
Fixed